cli/cmd/cluster-apply.go

```diff
@@ -106,14 +106,19 @@ func runClusterApply(cmd *cobra.Command, args []string) {
 			ex:         *ex,
 		}
 
-		releases := []string{"pod-checkpointer", "kube-apiserver", "kubernetes", "calico"}
+		releases := map[string][]string{
```

Comment:
I think we should actually have a slice with namespaces here, to make sure the order in which we upgrade things is deterministic.